src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -81,6 +81,19 @@ public UniquePersonList(UniquePersonList source) {
         return Collections.unmodifiableList(internalList);
     }
 
+    /**
+     * Similar to immutableListView.
+     * Returns an unmodifiable java List view with elements cast as immutable {@link ReadOnlyPerson}s.
+     * For use with other methods/libraries.
+     * Any changes to the internal list/elements are immediately visible in the returned list.
+     * This list has the persons sorted by name in alphabetical order.
+     */
+
```

Comment:
Good effort at writing a comprehensive header comment. However, some information such as `For use with other methods/libraries` seem to be redundant, so let's avoid those to keep header comments short and readable (i.e. containing only important information)

P/S There's also a redundant spacing here :0 and later in line 96